android/app/src/main/java/com/microsoft/codepush/react/CodePushConstants.java

```diff
@@ -0,0 +1,28 @@
+package com.microsoft.codepush.react;
+
+public class CodePushConstants {
+    static final String ASSETS_BUNDLE_PREFIX = "assets://";
```

Comment:
Should we explicitly mark all of these are public? I believe we chose to be explicit in the Windows implementation, and RN Java code seems to do the same.